templates/matchingDropDown.jsx

```diff
@@ -176,16 +177,19 @@ export default function MatchingDropDown(props) {
           !_isEnabled && 'is-disabled',
           hasActiveOption && 'is-selected'
         ])}
+        role="combobox"
         aria-haspopup="listbox"
         aria-expanded={isOpen}
+        aria-controls={`${_id}-matching-item-${_itemIndex}__list`}
         onMouseDown={onStartInteraction}
         onTouchStart={onStartInteraction}
         onClick={onButtonClick}
         ref={button}
-        aria-labelledby={props.questionTitleId + ' ' + props.questionTextId}
+        aria-labelledby={props.ariaLabel}
```

Comment:
Apologies this was missed. Fixed in https://github.com/adaptlearning/adapt-contrib-matching/commit/d7efd805b2d57a483fadea34120730371fe4043b

I've retested and the button reads as per the following:

- If `text`, this sets the `aria-labelledby`
- otherwise, if `ariaQuestion`, this sets the `aria-label`
- otherwise `displayTitle body instruction` sets the `aria-labelledby`